docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -44,6 +48,7 @@ The date must be a hyphen-separated ISO8601 date in the format of:
  * a two digit day.
 
 For example:
+
```

Comment:
This blank line makes the rendering of bullet points consistent across this document. Surprisingly the existence of this blank line changes how the rendered html appears...